### PR TITLE
experiment: typescript

### PR DESCRIPTION
**Pull request by Jamalam360**

### CODE REVIEW

**Comment by Jamalam360** [comment]
should be no functional changes

File: src/routes/issues/+page.svelte [hunk 1]
```diff
@@ -45,37 +30,48 @@
 />
 
 <div style="display: flex;">
+	<!-- TODO: Use a custom component for this, I don't think we should need a library for it -->
```
Comment (PugzAreCute) [inline src/routes/issues/+page.svelte:33]: I agree, but I heard that this has better performance somewhere

**Review by PugzAreCute — APPROVED** [review]
Seems fine :)

File: src/routes/issues/+page.svelte (hunk 1 above)
Comment (Jamalam360) [reply src/routes/issues/+page.svelte:33]: Alright, we'll look into it later

**Comment by Jamalam360** [comment]
For the sake of getting the ball rolling again, this pull request will be merged tomorrow if no complaints are brought forward

**Pull request merged**